components/dropbox/sources/all-updates/all-updates.mjs

```diff
@@ -1,25 +1,35 @@
-const dropbox = require("../../dropbox.app.js");
+import dropbox from "../../dropbox.app.mjs";
 
-module.exports = {
+export default {
   key: "dropbox-new-or-modified-file-or-folder",
   name: "New or Modified File or Folder",
-  version: "0.0.4",
-  description:
-    "Emits an event when a file or folder is added or modified. Make sure the number of files/folders in the watched folder does not exceed 4000.",
+  version: "0.0.5",
+  description: "Emits an event when a file or folder is added or modified. Make sure the number of files/folders in the watched folder does not exceed 4000.",
   props: {
     dropbox,
-    path: { propDefinition: [dropbox, "path"] },
-    recursive: { propDefinition: [dropbox, "recursive"] },
+    path: {
+      propDefinition: [
+        dropbox,
+        "pathFolder",
+        () => ({
+          returnSimpleString: true,
+        }),
+      ],
+    },
+    recursive: {
+      propDefinition: [
+        dropbox,
+        "recursive",
+      ],
+    },
```

Comment:
If you create a `common.mjs` file in sources and include `...common.props` within the props section you can remove props `dropbox`, `db`, `path`, and `recursive`, & `dropboxApphook`.